fastparquet/test/test_writer.py

```diff
@@ -506,3 +506,141 @@ def test_many_categories(tempdir, n):
     out = pf.to_pandas(categories=['x'])
 
     tm.assert_frame_equal(df, out)
+
+
+@pytest.mark.parametrize('row_groups', ([0], [0, 2]))
+@pytest.mark.parametrize('dirs', (['', ''], ['cat=1', 'cat=2']))
```

Comment:
The equals sign is not a reserved character for windows, apparently https://msdn.microsoft.com/en-ca/library/windows/desktop/aa365247(v=vs.85).aspx